.github/workflows/rust-workflow.yml

```diff
@@ -37,37 +35,39 @@ jobs:
 
     - name: Caching
       uses: Swatinem/rust-cache@b8a6852b4f997182bdea832df3f9e153038b5191
-      with:
-        workspaces: "head -> target"
 
     - name: Rustfmt
       shell: bash
-      working-directory: head
       run: cargo +${NIGHTLY_VERSION} fmt -- --check
 
     - name: Clippy
-      working-directory: head
       shell: bash
       run: cargo +${NIGHTLY_VERSION} clippy --workspace --all-targets -- -D warnings
     
-    - name: Test
-      timeout-minutes: 20
-      shell: bash
-      working-directory: head
-      run: cargo nextest run --workspace
-
     - name: Doc-test
       shell: bash
-      working-directory: head
       run: cargo +${NIGHTLY_VERSION} test --doc --workspace --verbose
 
     - name: Udeps
       shell: bash
-      working-directory: head
       run: cargo +${NIGHTLY_VERSION} udeps
 
     - name: Typos
       shell: bash
-      working-directory: head
       run: typos
 
+    - name: Test
+      timeout-minutes: 20
```

Comment:
Do we need this? Nextest has more granular features for setting test timeouts, and sensible defaults.

Reply:
remoeved